ohmec_data_na.js

```diff
@@ -164,6 +164,145 @@ dataNA = {
         "type":"Polygon",
         "coordinates":[
           [[-76.68474,36.0],[-121.534,36.0],[-121.332757,35.784382],[-121.283465,35.674843],[-121.168449,35.636505],[-121.004141,35.461243],[-120.905556,35.450289],[-120.894602,35.247642],[-120.765783,35.163671],[-120.702839,35.172158],[-120.631709,35.099764],[-120.670048,34.902595],[-120.609801,34.858779],[-120.638616,34.758616],[-120.604931,34.710816],[-120.64814,34.579455],[-120.531601,34.53526],[-120.472878,34.448009],[-120.138784,34.475393],[-119.875891,34.40967],[-119.558229,34.415147],[-119.278905,34.26727],[-119.218659,34.146777],[-118.802411,33.998899],[-118.566903,34.042715],[-118.391641,33.840068],[-118.413548,33.741483],[-118.260194,33.703145],[-118.183517,33.763391],[-117.7837,33.538836],[-117.471515,33.297851],[-117.329114,33.122589],[-117.252437,32.876127],[-117.24696,32.668003],[-117.126467,32.536556],[-117.16376,32.482743],[-117.037417,32.269337],[-116.933047,32.250756],[-116.883608,31.994884],[-116.603457,31.85035],[-116.641909,31.738297],[-116.735293,31.756982],[-116.641909,31.663519],[-116.691347,31.574643],[-116.301333,31.157199],[-116.33974,31.0],[-114.780412,31.0],[-114.856631,31.067842],[-114.856631,31.448199],[-114.746767,31.677545],[-114.614931,31.756982],[-114.104067,31.476312],[-113.950258,31.593361],[-113.609682,31.476312],[-113.631655,31.326276],[-113.043886,31.1619],[-113.100857,31.0],[-81.414176,31.0],[-81.400385,31.13446],[-81.290846,31.20566],[-81.271641,31.258666],[-81.279893,31.364491],[-81.175831,31.517845],[-81.130598,31.632375],[-81.132015,31.693108],[-80.992095,31.858222],[-80.851039,31.988185],[-80.843186,32.023769],[-80.885553,32.032678],[-80.660998,32.246279],[-80.452875,32.328433],[-80.430967,32.399633],[-80.206412,32.552987],[-79.998289,32.613234],[-79.866842,32.755634],[-79.631334,32.887081],[-79.582041,33.007573],[-79.357487,33.007573],[-79.187701,33.171881],[-79.149363,33.380005],[-78.935762,33.637421],[-78.716684,33.80173],[-78.541422,33.851022],[-78.179944,33.916745],[-77.971821,33.845545],[-77.82942,34.163208],[-77.555573,34.415147],[-77.210526,34.60684],[-76.991448,34.667086],[-76.673786,34.694471],[-76.493047,34.661609],[-76.279446,34.940933],[-76.394462,34.973795],[-76.536862,35.14358],[-76.482093,35.313365],[-76.147999,35.324319],[-75.89606,35.570781],[-75.775568,35.581735],[-75.715321,35.696751],[-75.781044,35.937736],[-75.961783,35.899398],[-76.060368,35.992506],[-76.3616,35.943213],[-76.399939,35.987029],[-76.673786,35.937736],[-76.68474,36.0]]]}},
+    { "type":"Feature",
+      "id":"MD1",
```

Comment:
Trying to keep the database sorted by startdatestr; MD1 erroneously was slotted later; no material changes to the entry